Sources/Classes/Helpers/Platforms/iOS/RSiOSLifecycleEvents.swift

```diff
@@ -91,4 +96,64 @@ class RSiOSLifecycleEvents: RSPlatformPlugin, RSiOSLifecycle {
     }
 }
 
+extension RSiOSLifecycleEvents {
+    func getApplicationUpdatedProps(previousVersion: String?, previousBuild: String?, currentVersion: String?, currentBuild: String?) -> [String: Any] {
+        var properties = [String: Any]()
+        if let previousVersion = previousVersion, !previousVersion.isEmpty {
```

Comment:
`!previousVersion.isEmpty`
Fo this you can extension of String like:
```
extension String {
  var isNotEmpty: Bool {
    return !isEmpty
  }
}
```
Use it like `previousVersion.isNotEmpty`